sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobContainerAsyncClient.java

```diff
@@ -1075,8 +1098,26 @@ public String generateUserDelegationSas(BlobServiceSasSignatureValues blobServic
      * @return A {@code String} representing all SAS query parameters.
      */
     public String generateSas(BlobServiceSasSignatureValues blobServiceSasSignatureValues) {
+        return generateSas(blobServiceSasSignatureValues, Context.NONE);
+    }
+
+    /**
+     * Generates a service SAS for the container using the specified {@link BlobServiceSasSignatureValues}
+     * Note : The client must be authenticated via {@link StorageSharedKeyCredential}
```

Comment:
This will generate on the same line as the sentence above due to Javadocs being HTML, let's make this another `<p>` so get a line break.

Reply:
Fixed